examples/hello/README.md

```diff
@@ -3,3 +3,21 @@
 Our interactive Hello World. Also a good template for your first own app. [This article](http://eclipsesource.com/blogs/2015/02/20/tabris-js-examples-hello-world/) discusses the example in detail.
 
 ![Hello world](https://tabrisjs.com/assets/public-content/img/examples/hello-world-android-ios.png)
+
+## Running this example
```

Comment:
Can you please move the _Running this example_ part to examples/README.md and adjust it accordingly? It applies to all examples.